packages/docs-app/src/common/films.tsx

```diff
@@ -132,15 +132,27 @@ export const TOP_100_FILMS: IFilm[] = [
     { title: "Monty Python and the Holy Grail", year: 1975 },
 ].map((m, index) => ({ ...m, rank: index + 1 }));
 
-export const renderFilm: ItemRenderer<IFilm> = (film, { handleClick, handleFocus, modifiers, query }) => {
+export const renderFilm: ItemRenderer<IFilm> = (
+    film: IFilm,
+    {
+        handleClick,
+        handleFocus,
+        modifiers,
+        query,
+    }: {
+        handleClick: () => void;
```

Comment:
you don't need any of these type defs. they are inferred through `: ItemRenderer<IFilm`. please remove them, as this code is now overly verbose.